### PR TITLE
Cnft1 2492 expand more

### PR DESCRIPTION
**Pull request by alaapbharadwaj**
## Description

"View more" and "View less" buttons added to column data for multi valued columns.
Screenshot 2024-07-31 at 2 48 23 PM

Screenshot 2024-07-31 at 2 48 34 PM

Also snuck in a small bug fix with the Outbreak name chip, so it is displaying as "OUTBREAK NAME"
## Tickets

* [CNFT1-2492](https://cdc-nbs.atlassian.net/browse/CNFT1-2492)

## Checklist before requesting a review
- [ ] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

[CNFT1-2492]: https://cdc-nbs.atlassian.net/browse/CNFT1-2492?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/modernization-ui/src/components/Search/ExpandableResults.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    PatientSearchResultAddress,
+    PatientSearchResultIdentification,
+    PatientSearchResultName
+} from 'generated/graphql/schema';
+import { useState } from 'react';
+import styles from './expandableResults.module.scss';
+
+type Props = {
+    results: PatientSearchResultAddress[] | PatientSearchResultIdentification[] | PatientSearchResultName[] | string[];
+    renderResult?: ({ _ }: any) => string;
```
Comment (mpeels) [inline apps/modernization-ui/src/components/Search/ExpandableResults.tsx:12]: Issue: Use of `any`

Comment (alaapbharadwaj) [reply apps/modernization-ui/src/components/Search/ExpandableResults.tsx:12]: Talked to adam and working on refactoring the whole component so it is a feature of the DataTable by changing the row height when View more is clicked

File: apps/modernization-ui/src/design-system/table/HeightClamp.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+};
+
+type ClampState = 'constrained' | 'expanded' | 'simple';
+
+export const HeightClamp = ({ children }: Props) => {
+    const measureRef = useRef<HTMLTableRowElement | null>(null);
+    const [heightState, setState] = useState<ClampState>('simple');
+
+    useEffect(() => {
+        console.log(measureRef.current?.clientHeight);
```
Comment (hclarkEnq) [inline apps/modernization-ui/src/design-system/table/HeightClamp.tsx:17]: log

**Review by hclarkEnq — DISMISSED** [review]

File: apps/modernization-ui/src/components/Search/ExpandableResults.tsx
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/components/Search/ExpandableResults.tsx:1]: question: Is this file still relevant?  did `HeightConstrained` take it's job?

**Review by hclarkEnq — APPROVED** [review]

**Pull request merged**